app/views/likes/show_dislikes.html.erb

```diff
@@ -0,0 +1,5 @@
+<% Post.find(params[:id]).likes.where(vote: false).each do |x|  %>
```

Comment:
Too much logic in views, try using methods from controllers, models or view helpers.